src/openrct2/ride/Ride.cpp

```diff
@@ -4936,7 +4936,7 @@ static bool ride_create_cable_lift(ride_id_t rideIndex, bool isApplying)
     if (ride == nullptr)
         return false;
 
-    if (ride->mode != RideMode::ContinuousCircuitBlockSectioned && ride->mode != RideMode::ContinuousCircuit)
+    if (!(ride->IsBlockSectioned()))
```

Comment:
This statement contains superfluous parentheses. I suggest:
```suggestion
    if (!ride->IsBlockSectioned())
```